Jenkinsfile-smoke.groovy

```diff
@@ -0,0 +1,16 @@
+/*
+ * Requires: https://github.com/RedHatInsights/insights-pipeline-lib
+ */
+
+@Library("github.com/RedHatInsights/insights-pipeline-lib") _
+
+
+if (env.CHANGE_ID) {
+    runSmokeTest (
+        ocDeployerBuilderPath: "approval",
+        ocDeployerComponentPath: "approval",
+        ocDeployerServiceSets: "approval",
```

Comment:
Does approval interact with any other service sets that you will want to test?

Reply:
yes. rbac.

Reply:
Right, ok. And that's not in e2e-deploy yet... we can work on that this week. So I guess to get things started you can just deploy approval ;)

Reply:
@zsadeh What about kafka?  I see the catalog version includes `platform-mq`

See https://github.com/ManageIQ/catalog-api/pull/308/files

cc @bzwei

Reply:
`approval-api` has dependencies on `kafka` and `approval-process`, but they are optional. It depends on what are included in the smoke tests. If `kafka` is not present errors are logged but the request will not fail. If `approval-process` is not present, you cannot test email notification.

We however need to consider the code promotion process for `approval-process`. Let's do it after this work.

Reply:
At the moment, since `approval` is listed as a service set to deploy, the whole `approval` service set will be deployed, which includes all the components here: https://github.com/RedHatInsights/e2e-deploy/tree/master/templates/approval

Reply:
@bsquizz Thanks for the clarification